### PR TITLE
LAMP_LV_17 motion class addition to lamp_motion.py

### PR DESCRIPTION
**Pull request by Mbosum**
## Description

Added LAMPFlowCell class at the end of lamp_motion.py to connect to motor PVs associated with the x454 configuration of LAMP motion components of the LAMP IP1 endstation.

## Motivation and Context

In addition to the Gas Jet, and Sample Paddle manipulators, a 2 axis Detector manipulator has been added for the LV17 configuration of LAMP. These axes need to be reflected in the motion control GUI.

## How Has This Been Tested?

I tested the module with flake8 it came back clean:

Screenshot (498)

I also tested whether I can make a device with it to verify the PV and component names:
Screenshot (499)

## Where Has This Been Documented?

There is a docstring for the class.

Confluence documentation is found here:
https://confluence.slac.stanford.edu/pages/viewpage.action?pageId=337052991

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [ ] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [ ] Test suite passes on travis
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: docs/source/upcoming_release_notes/933-lamp_lv_17.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+-----------
+- N/A
+
+Bugfixes
+--------
+- N/A
+
+Maintenance
+-----------
+- Mobosum
```
Comment (klauer) [inline docs/source/upcoming_release_notes/933-lamp_lv_17.rst:26]:
```suggestion
- N/A
```

Your name just goes below in Contributors :)

**Review by klauer — APPROVED** [review]
I think this looks good, pending test results. You did the best you could with the class name, (considering how unwieldy it is even in just regular text - LAMP LV 17...).

**Review by ZLLentz — APPROVED** [review]
LGTM aside from the accident in the pre-release notes

File: docs/source/upcoming_release_notes/933-lamp_lv_17.rst (hunk 1 above)
Comment (ZLLentz) [reply docs/source/upcoming_release_notes/933-lamp_lv_17.rst:26]: I guess I'll fix this when we make the tag...

**Pull request merged**